less/_grid.less

```diff
@@ -181,6 +181,37 @@
 				padding-bottom: 0;
 			}
 		}
+	}
+
+	// 3 COLUMNS
+	[class*='col'] {
+		@media @mq-md {
+			.sss&,
```

Comment:
The key for these new classes is `s = skinny` and `w = wide`. These refer to the column layout.

Let's look at **three column** layouts
- First two columns have width of `.col3`, last column has width of `.col6`. Use `.ssw` class
- First column has width of `.col8`, last two columns have width of `.col2`. Use `.wss` class
- All three columns have width of `.col4`. Use `.sss` class

Now a **four column** layout
- All columns same width of `.col3`. Use `.ssss` class